src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -99,8 +136,8 @@ public static void PrintHelp(Network network)
         {
             var builder = new StringBuilder();
 
-            builder.AppendLine($"-apiuri=<string>                  URI to node's API interface. Defaults to '{ DefaultApiHost }'.");
-            builder.AppendLine($"-apiport=<0-65535>                Port of node's API interface. Defaults to { network.DefaultAPIPort }.");
+            builder.AppendLine($"-apiuri=<string>                  URI to node's API interface. Defaults to '{ settingsHelp.ApiUri }'.");
```

Comment:
This is a bit misleading, the default has to come from Network.

Reply:
Fixed